ansible.cfg

```diff
@@ -12,5 +12,8 @@ internal_poll_interval = 0.001
 command_warnings = False
 deprecation_warnings = False
 
+# uncomment to speed up the install a bit by squashing package installations
+#squash_actions = apk,apt,dnf,homebrew,pacman,package,pkgng,yum,zypper
```

Comment:
This is new and exciting and I would like to subscribe to your newsletter. 

Under what circumstances would we _not_ want to squash package installations?

Reply:
@whikloj it's a "controversial" option.  There's talk of it even being deprecated in ansible 2.11.. I'm saddened by that, but certain packagers can't handle it right.  So if you use the `package` module to install packages instead of the OS specific manager modules, it *may* not work right.  I figured that disabling it by default would be a better option than fielding the questions about why things were failing if it were deprecated or what have you.  Kind of a "if you know about it, here it is" option, but a little more forward by including it in the ansible.cfg in the repo.

Reply:
Another thought is that it is disabled by default in the stock ansible.cfg, that was another reason for not enabling it by default.

Reply:
Ohhh ok, that makes sense.